helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloDeployment.java

```diff
@@ -349,9 +319,7 @@ private String getHostPort(final String containerId, final int containerPort)
       final NetworkSettings settings = dockerClient.inspectContainer(containerId).networkSettings();
       for (Map.Entry<String, List<PortBinding>> entry : settings.ports().entrySet()) {
         if (entry.getKey().equals(heliosPort)) {
-          for (PortBinding portBinding : entry.getValue()) {
-            return portBinding.hostPort();
-          }
+          return entry.getValue().get(0).hostPort();
```

Comment:
Just style. I've seen `for (..) { return ..}` far less than the other. It can be a little confusing to see a for loop that iterates exactly once.

You can also do `return entry.getValue().iterator().next() ...` which is what the `for (..) { return }` variant basically does anyway.